From 5365564901778d96a81e00e34c804d4fb05f0093 Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Tue, 30 Mar 2021 14:42:05 +0900
Subject: [PATCH] cifsd: fix wrong prototype in comment
Git-commit: 5365564901778d96a81e00e34c804d4fb05f0093
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


kernel test robot reported:
 >> fs/cifsd/oplock.c:1454: warning: expecting prototype for
    create_durable_rsp__buf(). Prototype was for
    create_durable_rsp_buf() instead

This patch fix wrong prototype in comment.

Reported-by: kernel test robot <lkp@intel.com>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/cifsd/oplock.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/cifsd/oplock.c b/fs/cifsd/oplock.c
index 8e072c3e7b89..4ff23aee69fa 100644
--- a/fs/cifsd/oplock.c
+++ b/fs/cifsd/oplock.c
@@ -1436,7 +1436,7 @@ struct create_context *smb2_find_context_vals(void *open_req, const char *tag)
 }
 
 /**
- * create_durable_rsp__buf() - create durable handle context
+ * create_durable_rsp_buf() - create durable handle context
  * @cc:	buffer to create durable context response
  */
 void create_durable_rsp_buf(char *cc)
-- 
2.33.0

